keps/sig-network/563-dual-stack/README.md

```diff
@@ -1299,3 +1306,54 @@ This capability will move to stable when the following criteria have been met.
 * e2e test successfully running on two platforms
 * testing ingress controller infrastructure with updated dual-stack services
 * dualstack tests run as pre-submit blocking for PRs
+
+
+### Feature Enablement and Rollback
+
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [X] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: IPv6DualStack
+    - Components depending on the feature gate: api-server, controller-manager, and kubelet
```

Comment:
[nit] Please use the following names for consistency
      - kube-apiserver
      - kube-controller-manager
      - kubelet
      - kube-proxy